app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -134,11 +146,13 @@
     <string name="hdf" translation_description="hubble object">哈伯深場影像</string>
     <string name="rho_oph" translation_description="hubble object">心宿增四星</string>
     <string name="menu_tos">服務條款</string>
+    <string name="menu_diagnostics">Diagnostics</string>
```

Comment:
This shouldn't be needed - if we don't have a Chinese translation for this text it should already fall back to English.  Did you mean to provide a translation here?

Reply:
@jaydeetay, I will correct it and add new commit to this pull request. thank you.